__tests__/test.js

```diff
@@ -45,3 +105,79 @@ describe('getOrThrow', () => {
         expect(getOrThrow('TIMEZONE')).to.equal(process.env.TIMEZONE);
     });
 });
+
+describe('createIdObject', () => {
+    it('should create a object id', () => {
+        const returnObject = createIdObject(
+            ['mockedUser', 'mockedUser@user.com'],
+            mockedContributed
+        );
+
+        expect(returnObject.alternateId).to.equal('mockedUser@user.com');
+        expect(returnObject.github).to.equal('mockedUser');
+
+        const contributionsObject = [
+            { id: '1', type: 'IssueCommentEvent' },
+            { id: '2', type: 'PullRequestEvent' },
+        ];
+
+        contributionsObject.forEach((contribution, index) => {
+            expect(returnObject.contributions[index].id).to.equal(contribution.id);
+            expect(returnObject.contributions[index].type).to.equal(contribution.type);
+        });
+    });
+});
+
+describe('parseDatesFromArgv', () => {
+    beforeEach(() => {
+        process.env = {
+            GITHUB_TOKEN: 'mockToken',
+            TIMEZONE: 'America/Los_Angeles',
+            CSV_COLUMN_NUMBER_FOR_GITHUB_ID: '0',
+            CSV_COLUMN_NUMBER_FOR_ALTERNATE_ID: '1',
+            GITHUB_IMPORTANT_EVENTS:
+                'CommitCommentEvent,IssueCommentEvent,IssuesEvent,PullRequestEvent,PullRequestReviewEvent,PullRequestReviewCommentEvent',
+        };
+
+        process.argv[2] = '2020-01-01';
+        process.argv[3] = '2020-12-01';
+    });
+
+    it('should generate 2 dates based on arguments', () => {
+        const moments = parseDatesFromArgv();
+
+        expect(`${moments[0]}`).to.equal('Wed Jan 01 2020 00:00:00 GMT-0800');
+        expect(`${moments[1]}`).to.equal('Tue Dec 01 2020 23:59:59 GMT-0800');
+
+        expect(`Users that contributed between ${moments[0]} and ${moments[1]}`).to.equal(
+            'Users that contributed between Wed Jan 01 2020 00:00:00 GMT-0800 and Tue Dec 01 2020 23:59:59 GMT-0800'
+        );
+    });
+});
+
+describe('filterContributorByTime', () => {
+    beforeEach(() => {
+        process.env = {
+            GITHUB_TOKEN: 'mockToken',
+            TIMEZONE: 'America/Los_Angeles',
```

Comment:
Same question here about daylight saving time potentially messing with the test results.